docs/monitoring/spm-faq.md

```diff
@@ -571,24 +575,65 @@ The resulting package will contain all relevant info needed for our
 investigation. You can send it, along with short description of your
 problem, to <support@sematext.com> or contact us in chat.
 
+<b>Container setups</b>
+
+At the moment there is no diagnostics script for container setups, so various resources have to be gathered manually.
+
+Two types of agents are running in container setups and spawn from the following images:
+  - <b>sematext/agent</b> (also known as STA) - collects infrastructure data (OS and container info, metrics and events)
+  - <b>sematext/app-agent</b> (known as application agent or AA) - collects application metrics (e.g. metrics of your Elasticsearch or Kafka).
+    For each monitored service instance one such application agent container will exist. These containers are created automatically by STA.
+
+<b>sematext/agent</b>
+
+Docker:
+
+```
+sudo docker logs st-agent > /tmp/st-agent.log 2>&1 && tar -cvzf /tmp/st-agent.tgz /tmp/st-agent.log
+```
+
+Swarm:
+
+```
+sudo docker service logs st-agent > /tmp/st-agent.log 2>&1 && tar -cvzf /tmp/st-agent.tgz /tmp/st-agent.log
+```
+
+Kubernetes:
+
+This will produce multiple tgz files in /tmp dir, one for each running sematext/agent instance:
+
+```
+for pod in $(kubectl get --no-headers -o=custom-columns=NAME:.metadata.name pods --selector=app=sematext-agent);do kubectl logs $pod -c agent | gzip > /tmp/st-agent-$pod.gz;done
```

Comment:
Do we think these .gz files could ever be big?  If yes, do we want to use gzip -9?

Reply:
Potentially could be very big, will add -9.